packages/react-scripts/config/rangle/postcss.js

```diff
@@ -0,0 +1,32 @@
+const webpack = require('webpack');
+
+const postcssBasePlugins = [
+  require('postcss-modules-local-by-default'),
+  require('postcss-import')({
+    addDependencyTo: webpack,
+  }),
+  require('postcss-cssnext')({
+    browsers: [
```

Comment:
If you omit this spec in the cssnext config you can add a `browserslist` file in the project root which can contain this same version info and cssnext can pick it up.  Maybe we can add a default browserslist file instead of hard coding it?

That way client projects can tune without needed to `eject`?